app/templates/components/chart-toggle.html

```diff
@@ -0,0 +1,18 @@
+<p class="text-corporate-blue text-center text-h2">
+  {{ toggleText }}
+  <select
+    id="{{ section_id }}_toggle"
+    class="bg-white text-center font-bold border-b-2 border-corporate-blue inline"
+    aria-controls="{{ section_id }}_ber {{ section_id }}_ger"
+    aria-label="Schalte um zwischen Berlin und Deutschland"
```

Comment:
This should probably also come from `translations` in the end? (Just mentioning this here, in case it was forgotten.)

Reply:
Absolutely